src/Api/Controllers/UserLinkedSourcesController.php

```diff
@@ -86,21 +86,14 @@ public function devices() : Response
                 $deviceId = $this->validateAppleUser($deviceId)->sub;
             }
 
-            UserLinkedSources::updateOrCreate([
-                'conditions' => 'users_id = ?0 AND source_users_id_text = ?1 AND source_id = ?2',
-                'bind' => [
-                    $this->userData->getId(),
-                    $deviceId,
-                    $source->getId()
-                ]
-            ], [
-                'users_id' => $this->userData->getId(),
-                'source_id' => $source->getId(),
-                'source_users_id' => $this->userData->getId(),
-                'source_username' => $this->userData->displayname . ' ' . $app,
-                'is_deleted' => 0,
-                'source_users_id_text' => $deviceId,
-            ]);
+            $userLinkedSource = new UserLinkedSources();
```

Comment:
@rwhite27 dont we still need to confirm if the source_users_id_text exist to avoid saving it over and over again?